manifests/q/Quadren/Arc/Prerelease/PBv1.1.3/Quadren.Arc.Prerelease.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Quadren.Arc.Prerelease
+PackageVersion: PBv1.1.3
+PackageLocale: en-US
+Publisher: Quadren
+PackageName: Arc
+License: MIT License
+ShortDescription: Arc Seamless File Transfer

```

```suggestion
ShortDescription: Arc is a seamless cross-platform file transfer service by Quadren. Fast, secure and 100% free, with support for Windows, MacOS, Android and iOS.
```
ShortDescription should be descriptive and meaningful